docs/reference/client-side-encryption.txt

```diff
@@ -324,6 +324,203 @@ master key and create data keys, see the following sections of this tutorial:
 - `Creating A Master Key`_,
 - `Creating A Data Key`_,
 
+Queryable Encryption
+====================
+Queryable encryption is a new feature in MongoDB 6.0. It also requires
+libmongocrypt version 1.5.0-rc1 or above.
+
+Usage of automatic queryable encryption is very similar to previously
+known client-side Encryption. Below is an example of automatic queryable
+encryption.
+
+.. code-block:: ruby
+
+  require 'mongo'
+
+  #####################################
+  # Step 1: Create a local master key #
+  #####################################
+
+  # A local master key is a 96-byte binary blob.
+  local_master_key = SecureRandom.random_bytes(96)
+  # => "\xB2\xBE\x8EN\xD4\x14\xC2\x13\xC3..."
+
+  #############################
+  # Step 2: Create a data key #
+  #############################
+
+  kms_providers = {
+    local: {
+      key: local_master_key
+    }
+  }
+
+  # The key vault client is a Mongo::Client instance connected to the collection
+  # that will store your data keys.
+  key_vault_client = Mongo::Client.new(['localhost:27017'])
```

Comment:
This should specify the `keyvault` database I think as the default is `admin`.

Reply:
Or, if this does not matter, remove "connected to the collection" from the above text.